src/components/Search/Search.test.tsx

```diff
@@ -155,7 +155,7 @@ describe('<Search /> component test', () => {
     beforeEach(() => {
       jest.resetModules();
       jest.doMock('lodash/debounce', () => {
-        return function debounceMock(fn) {
+        return function debounceMock(fn: unknown) {
```

Comment:
Please are you sure about this type?

Reply:
Since this is only a type to satisfy the unit test and called internally by jest aka never called manually by production code I just used the fastest route to satisfy the compiler.

Reply:
I see...but maybe it's better to right a function type, since this is a function right? for example: `fn: () => void`. some investigation might be need to check what we could write to replace void..

Reply:
Maybe just a `Function` type would be also fine